app/views/content/funding-and-support/if-youre-disabled.md

```diff
@@ -31,11 +31,21 @@ quote:
     text: "Growing up as a profoundly deaf child, I had a positive upbringing in my mainstream primary school. The teachers were amazing in terms of support and treating me equally. They inspired me to become a primary school teacher."
     name: "Habiba, primary school teacher"
     classes: quote--indent
+cta_arrow_link:
+  dsa:
+    link_target: "https://www.gov.uk/disabled-students-allowance-dsa"
+    link_text: "Find out if you're eligible for a bursary or scholarship"
+  access-to-work:
+    link_target: "https://www.gov.uk/access-to-work"
+    link_text: "Find out more about Access to Work"
+  adjustments:
+    link_target: "https://www.gov.uk/access-to-work"
+    link_text: "Find out more about adjustments"
```

Comment:
```suggestion
    link_text: "Find out more about adjustments to help you apply and train"
```
Worth adding more context?

Also - presume this will go to a new page in Train to be a teacher (when added to PR!)?